bom/runtime/pom.xml

```diff
@@ -176,7 +176,7 @@
         <quarkus-spring-api.version>5.2.Final</quarkus-spring-api.version>
         <quarkus-spring-data-api.version>2.1.Final</quarkus-spring-data-api.version>
         <quarkus-spring-security-api.version>5.2.Final</quarkus-spring-security-api.version>
-        <quarkus-spring-boot-api.version>2.1.Final</quarkus-spring-boot-api.version>
+        <quarkus-spring-boot-api.version>2.2.Alpha1</quarkus-spring-boot-api.version>
```

Comment:
Hm... Did we update the Spring Boot version? If so, why? If not, why did we bump the Spring Boot version here?

Reply:
Yes, I updated the spring boot version, I thought it would be ok but as talked before let's keep the 2.1.13